base/dotenv.default.mustache

```diff
@@ -0,0 +1,3 @@
+PORT=
+RETHINK_HOST=
+RETHINK_NAME=
```

Comment:
This file is used as the template to check which `env` vars you should have listed.